apps/i18n/common/en_us.json

```diff
@@ -2836,6 +2836,7 @@
   "standardsReportLessonLengthInfo": "*Lessons in this course offer between 45 and 65 minutes of instruction",
   "startBlocksTemplateWarning": "WARNING: You are editing start blocks for a level with a template. Start blocks should be defined in the template level.",
   "startCourse": "Start Course",
+  "startingPainter": "Starting painter.",
```

Comment:
Added here since `i18n/javalab/en_us.json` is not loaded for `pythonlab`. Should I remove the string from the javalab locale?